### ISSUE TITLE
Pre and Post hooks for simulations

### ISSUE THREAD

**Issue by eslick** [mhjort/clj-gatling#37]
In case this exists and it's a documentation issue, I'm finding several simulation settings where I want to configure or setup accounts on the server before running my scenarios, and then resetting the server afterwords for repeatability.  To properly test a mix of workloads against a single account, it's helpful to do this outside of a specific scenario so the scenarios, while operating more or less independently, are operating on a common set of data setup by the post hook.  e.g. 

```
(def simulation
   {:name "My Simulation"
    :pre-hook setup-fn
    :scenarios  [ ... ]
    :post-hook cleanup-fn})
```

**Comment by mhjort** [comment]
Makes sense. And this shouldn't be a big thing to implement. I'll try to do this in near future. Also PR is always welcome ;)

**Comment by mhjort** [comment]
@eslick This is now implemented in `0.9.0'
